core/control-plane/control-plane-aggregate-services/src/test/java/org/eclipse/edc/connector/service/query/QueryValidatorTest.java

```diff
@@ -82,12 +82,22 @@ void validate_interface_withoutSubtypeMap() {
     }
 
     @Test
-    void validate_isMapType() {
+    void validate_isMapTypeTrue() {
         queryValidator = new QueryValidator(TestObject.class);
         var query = with(criterion("someMap.foo", "=", "bar"));
 
         var result = queryValidator.validate(query);
 
+        assertThat(result.succeeded()).isTrue();
+    }
+
+    @Test
+    void validate_isMapTypeFalse() {
+        queryValidator = new QueryValidator(TestObject.class);
```

Comment:
Why does this set a field and not use a local var?

Reply:
This was existing code. Now maptype can or cannot be supported. I only changed method name and value to be passed. Rest was same as before. If you see other tests in that they all did not use local var..

Reply:
If there is no reason to use a field, please change it. The other code in that case is incorrect and there is no need to repeat the problem here.

Reply:
done